packages/flutter_driver/lib/src/extension.dart

```diff
@@ -123,7 +123,7 @@ class FlutterDriverExtension {
     return _onFrameReadyStream;
   }
 
-  Future<Health> getHealth(GetHealth command) async => new Health(HealthStatus.ok);
+  Future<Health> getHealth(Command command) async => new Health(HealthStatus.ok);
```

Comment:
should getHealth be private? It's not documented as only accepting a GetHealth command any more, but it will fail if given anything else.